app/autoload.php

```diff
@@ -2,6 +2,8 @@
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
 
+error_reporting(error_reporting() & ~E_USER_DEPRECATED);
```

Comment:
@Soullivaneuh there is a PR on symfony-standard to remove it. This was added in the 2.7.0 release when our deprecation warnings were not silenced.